pkg/vm/stack.go

```diff
@@ -170,6 +170,11 @@ func (s *Stack) Push(e Element) {
 	s.refs.Add(e.value)
 }
 
+// PushItem pushed an Item to the stack.
+func (s *Stack) PushItem(i stackitem.Item) {
```

Comment:
Don't we also want to replace all `PushVal` usages inside the `pkg/core/interop_system.go`?

Reply:
OK, let's try that too.